src/app/auth/auth.component.ts

```diff
@@ -201,6 +204,9 @@ export class AuthComponent implements OnInit, OnDestroy {
     const dataRepo: string = this.getDataRepoDetails(sessionInformation);
     this.githubService.storeOrganizationDetails(org, dataRepo);
 
+    this.logger.info("Selected Session: " + (this.sessionSelected || this.noSessionSelected));
+    this.logger.info("Selected Settings Location: " + sessionInformation);
```

Comment:
I think this single logging statement is sufficient (the settings location tells us the session).